### PR TITLE
Change wording for network requirements from recommendation to required

### PR DESCRIPTION
**Pull request by Martin-Weiss**
For VM migration, for storage replication and for VM access in production scenarios / scenarios under load - 1 GB will not be sufficient at all. Basically with 1 GB the VM migration will not complete if the VM is under load and the automatic and working VM migration will also be required for Harvester upgrades while VMs are up and running.

So we really should change "10 G recommended" to "10 G required".

### CODE REVIEW

**Review by bk201 — APPROVED** [review]
Thanks! @guangbochen we also need to adjust the [readme](https://github.com/harvester/harvester#hardware-requirements).

**Comment by guangbochen** [comment]
Sure I will update it later, @LucasSaintarbor can u PTAL and merge this one AEAP, thanks.

File: versioned_docs/version-v1.1/install/requirements.md [hunk 1]
```diff
@@ -18,11 +18,11 @@ Harvester nodes have the following hardware requirements and recommendations for
...
+| Memory           | 32 GB minimum for testing; 64 GB or above required for production                                                                                                                                                               |
+| Disk Capacity    | 200 GB minimum for testing; 500 GB or above required for production                                                                                                                                  |
 | Disk Performance | 5,000+ random IOPS per disk(SSD/NVMe). Management nodes (first 3 nodes) must be [fast enough for Etcd](https://www.ibm.com/cloud/blog/using-fio-to-tell-whether-your-storage-is-fast-enough-for-etcd) |
-| Network Card     | 1 Gbps Ethernet minimum for testing, 10Gbps Ethernet recommended for production                                                                                                                       |
+| Network Card     | 1 Gbps Ethernet minimum for testing, 10Gbps Ethernet minimum required for production                                                                                                                       |
```
Comment (LucasSaintarbor) [inline versioned_docs/version-v1.1/install/requirements.md:25]:
```suggestion
| Network Card     | 1 Gbps Ethernet minimum for testing, 10 Gbps Ethernet minimum required for production                                                                                                                       |
```

File: docs/install/requirements.md [hunk 2]
```diff
@@ -16,13 +16,13 @@ As an HCI solution on bare metal servers, there are minimum node hardware and ne
...
+| Memory           | 32 GB minimum for testing; 64 GB or above required for production                                                                                                                                     |
+| Disk Capacity    | 250 GB minimum for testing; 500 GB or above required for production                                                                                                                                   |
 | Disk Performance | 5,000+ random IOPS per disk(SSD/NVMe). Management nodes (first 3 nodes) must be [fast enough for Etcd](https://www.ibm.com/cloud/blog/using-fio-to-tell-whether-your-storage-is-fast-enough-for-etcd) |
-| Network Card     | 1 Gbps Ethernet minimum for testing, 10Gbps Ethernet recommended for production                                                                                                                       |
+| Network Card     | 1 Gbps Ethernet minimum for testing, 10Gbps Ethernet minimum required for production                                                                                                                  |
```
Comment (LucasSaintarbor) [inline docs/install/requirements.md:25]:
```suggestion
| Network Card     | 1 Gbps Ethernet minimum for testing, 10 Gbps Ethernet minimum required for production                                                                                                                  |
```

File: docs/install/requirements.md [hunk 3]
```diff
@@ -16,13 +16,13 @@ As an HCI solution on bare metal servers, there are minimum node hardware and ne
...
-| Disk Capacity    | 250 GB minimum for testing; 500 GB or above recommended for production                                                                                                                                  |
+| CPU              | x86_64 only. Hardware-assisted virtualization is required. 8-core processor minimum for testing; 16-core or above required for production                                                             |
+| Memory           | 32 GB minimum for testing; 64 GB or above required for production                                                                                                                                     |
+| Disk Capacity    | 250 GB minimum for testing; 500 GB or above required for production                                                                                                                                   |
 | Disk Performance | 5,000+ random IOPS per disk(SSD/NVMe). Management nodes (first 3 nodes) must be [fast enough for Etcd](https://www.ibm.com/cloud/blog/using-fio-to-tell-whether-your-storage-is-fast-enough-for-etcd) |
```
Comment (LucasSaintarbor) [inline docs/install/requirements.md:24]:
```suggestion
| Disk Performance | 5,000+ random IOPS per disk(SSD/NVMe). Management nodes (first 3 nodes) must be fast enough for etcd |
```

File: versioned_docs/version-v1.1/install/requirements.md [hunk 4]
```diff
@@ -18,11 +18,11 @@ Harvester nodes have the following hardware requirements and recommendations for
...
-| Disk Capacity    | 200 GB minimum for testing; 500 GB or above recommended for production                                                                                                                                  |
+| CPU              | x86_64 only. Hardware-assisted virtualization is required. 8-core processor minimum for testing; 16-core or above required for production                                                            |
+| Memory           | 32 GB minimum for testing; 64 GB or above required for production                                                                                                                                                               |
+| Disk Capacity    | 200 GB minimum for testing; 500 GB or above required for production                                                                                                                                  |
 | Disk Performance | 5,000+ random IOPS per disk(SSD/NVMe). Management nodes (first 3 nodes) must be [fast enough for Etcd](https://www.ibm.com/cloud/blog/using-fio-to-tell-whether-your-storage-is-fast-enough-for-etcd) |
```
Comment (LucasSaintarbor) [inline versioned_docs/version-v1.1/install/requirements.md:24]:
```suggestion
| Disk Performance | 5,000+ random IOPS per disk(SSD/NVMe). Management nodes (first 3 nodes) must be fast enough for etcd |
```

**Review by LucasSaintarbor — CHANGES_REQUESTED** [review]
Small nics and removed https://www.ibm.com/cloud/blog/using-fio-to-tell-whether-your-storage-is-fast-enough-for-etcd since the blog is no longer available.

**Comment by guangbochen** [comment]
> Small nics and removed https://www.ibm.com/cloud/blog/using-fio-to-tell-whether-your-storage-is-fast-enough-for-etcd since the blog is no longer available.

update to https://www.suse.com/support/kb/doc/?id=000020100 , thanks.

**Review by LucasSaintarbor — APPROVED** [review]
LGTM

**Pull request merged**